### DISCUSSION TITLE
Is this library subject to US EAR (Encryption and Export administration regulations)?

### DISCUSSION THREAD

**Discussion by ztest-dev** [rabbitmq/rabbitmq-server#9532] [Questions]
First, thank you so much for developing this open-source software which is powerful and feature-rich but also simple and easy to use. I prepare to use it in my projects. However, I wonder whether this software is subject to the Export Administration Regulations (EAR).
In case of being controlled by EAR, would you please tell me which classification number (ECCN) does it come under? Are there any restrictions or specific license requirements? May I know in which country or region did your team compile it? And does the software have encryption function? In case that encryption function is included, has the filing information been submitted according to EAR§742.15(b)?

I need this information to comply with the Export Administration Regulations (EAR), and I'm sure other users of this software would benefit from this information as well.

Thanks and best wishes！

**Comment by mkuratczyk** [comment]
https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/vmware-product-export-control-classification-list.pdf
